setup.cfg

```diff
@@ -10,20 +10,21 @@ classifiers =
     Development Status :: 3 - Alpha
     Environment :: Web Environment
     Framework :: Django
-    Framework :: Django :: 2.2
-    Framework :: Django :: 3.0
-    Framework :: Django :: 3.1
-    Framework :: Django :: 3.2
+    Framework :: Django :: 4.0
```

Comment:
There are others using Django pack on the platform that are likely on older versions. Do we need a comms and deprecation process to get folks to move to newer versions?

I'm mostly concerned that we're deprecating apps using older versions without alerting folks to move forward so if folks try to stage anything they won't successfully deploy.

Reply:
Since this package has different [released versions](https://pypi.org/project/cg-django-uaa/), if someone still needs to keep using an older version of Django, they can pin their dependency to an older release of this package

Reply:
👍  Just wanted to make sure we're not cutting folks off from using this without comms to the customers about what dependencies are going away and some lead time to make changes.